specs/workflows/steps/conflictResolution.spec.js

```diff
@@ -0,0 +1,379 @@
+			} );
+			expect( state.cr.newLines ).toEqual( [ "{", "	\"devDependencies\": {", "		\"@lk/some-package\": \"2.5.0\",", "		\"@lk/some-other-package\": \"1.3.0\",", "	}", "}" ] );
+			expect( state.cr.contents ).toEqual( `{
+	\"devDependencies\": {
+		\"@lk/some-package\": \"2.5.0\",
+	<<<<<<< HEAD
+		\"@lk/my-package\": \"14.14.0\",
+		\"@lk/my-other-package\": \"2.0.0\",
+	=======
+		\"@lk/my-package\": \"14.14.2-filterror.0\",
+		\"@lk/my-other-package\": \"blahblahblah\",
+	>>>>>>> f07c714... Bumped my-package to 14.14.2-filterror.0: conflicting change
+		\"@lk/some-other-package\": \"1.3.0\",
+	}
+}` );
+			expect( state.cr.localChanges ).toEqual( {
+				"my-package": "14.14.2-filterror.0",
+				undefined
+			} );
+		} );
+	} );
+
+	describe( "resolveChunkConflicts", () => {
+		it( "should update chunks with localChanges if locally changed packages are the ones in conflict", () => {
+			state = {
+				scope: "@lk",
+				cr: {
+					chunks: {
+						"		\"@lk/some-package\": \"2.5.0\",": [ "		\"@lk/my-perfect-package\": \"1.2.0\"," ]
+					},
+					localChanges: {
+						"my-perfect-package": "1.1.1-something.3"
+					}
+				}
+			};
+
+			run.resolveChunkConflicts( state );
+			expect( state.cr ).toHaveProperty( "chunks" );
+			expect( state.cr.chunks ).toEqual( {
+				"		\"@lk/some-package\": \"2.5.0\",": [ "		\"@lk/my-perfect-package\": \"1.1.1-something.3\"," ]
+			} );
+		} );
+
+		it( "should use HEAD changes with conflicted package in chunk isn't a pre-release", () => {
+			state = {
+				scope: "@lk",
+				cr: {
+					chunks: {
+						"		\"@lk/some-package\": \"2.5.0\",": [ "		\"@lk/my-package\": \"11.2.0\"," ]
+					},
+					localChanges: {
+						"my-perfect-package": "1.1.1-something.3",
+						"my-package": "11.2.0",
+						"some-random-package": "11.2.0"
+					}
+				}
+			};
+
+			run.resolveChunkConflicts( state );
+			expect( state.cr ).toHaveProperty( "chunks" );
+			expect( state.cr.chunks ).toEqual( {
+				"		\"@lk/some-package\": \"2.5.0\",": [ "		\"@lk/my-package\": \"11.2.0\"," ]
+			} );
+			expect( logger.log ).toHaveBeenCalledTimes( 1 );
+			expect( logger.log ).toHaveBeenCalledWith( "You had a local change of 11.2.0 for my-package, but we used HEAD's version of 11.2.0" );
+		} );
+
+		it( "should do nothing for with localChanges that aren't in conflicted chunks", () => {
+			state = {
+				scope: "@lk",
+				cr: {
+					chunks: {
+						"		\"@lk/some-package\": \"2.5.0\",": [ "		\"@lk/my-package\": \"11.2.0\"," ]
+					},
+					localChanges: {
+						"my-package": "11.2.0",
+						"some-random-package": "1.0.0"
+					}
+				}
+			};
+
+			run.resolveChunkConflicts( state );
+			expect( state.cr ).toHaveProperty( "chunks" );
+			expect( state.cr.chunks ).toEqual( {
+				"		\"@lk/some-package\": \"2.5.0\",": [ "		\"@lk/my-package\": \"11.2.0\"," ]
+			} );
+		} );
+
+		it( "should default version to [] when regex doesn't match localKey in conflict chunk", () => {
```

Comment:
Maybe update the `it` description because I keep wanting to look for an assert that looks for `[]`.